DfE.FindInformationAcademiesTrusts/Extensions/CategoriesOfConcernExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DfE.FindInformationAcademiesTrusts.Data;
+
+namespace DfE.FindInformationAcademiesTrusts.Extensions;
+
+public static class CategoriesOfConcernExtensions
+{
+    public static string ToDataSortValue(this CategoriesOfConcern rating)
+    {
+        return rating.ToDisplayString().ToLowerInvariant().Trim();
+    }
+
+    public static string ToDisplayString(this CategoriesOfConcern rating)
+    {
+        return rating switch
+        {
+            CategoriesOfConcern.None => "None",
+            CategoriesOfConcern.SpecialMeasures => "Special Measures",
+            CategoriesOfConcern.SeriousWeakness => "Serious Weakness",
+            CategoriesOfConcern.NoticeToImprove => "Notice To Improve",
+            _ => "Unknown"
```

Comment:
This is probably quite a likely scenario, "Unknown" should probably be "No data" or "Not applicable" lets chat to David in the am

Reply:
Ok, will bring it up in stand up

Reply:
This would only ever occur if the value was not defined in the `CategoriesOfConcern` enum so I think it is unlikely to occur